utils/generate_js_fixtures.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+import subprocess
+from pathlib import Path
+
+SPHINX_ROOT = Path(__file__).resolve().parent.parent
+TEST_JS_FIXTURES = SPHINX_ROOT / 'tests' / 'js' / 'fixtures'
+TEST_JS_ROOTS = SPHINX_ROOT / 'tests' / 'js' / 'roots'
+
+
+def build(srcdir: Path) -> None:
+    cmd = ('sphinx-build', '-E', '-q', '-b', 'html', f'{srcdir}', f'{srcdir}/_build')
```

Comment:
Can you use full options here ? It'd be a bit clearer in general (I think we use full options in CI/CD so that's why).